.ci/setup_env_fedora.sh

```diff
@@ -77,6 +72,21 @@ main()
 		echo "Install ${KATA_KSM_THROTTLER_JOB}"
 		chronic sudo -E dnf -y install ${KATA_KSM_THROTTLER_JOB}
 	fi
+
+	if [ "${TEST_CGROUPSV2}" == "true" ]; then
+		echo "Install podman dependencies"
+		sudo dnf -y builddep podman
+		dnf deplist podman --archlist "${arch}",noarch | awk '/provider:/ {print $2}' | sort -u | xargs sudo -E dnf -y install
+        	echo "Install podman"
+        	version=$(get_test_version "externals.podman.version")
+		podman_repo="github.com/containers/libpod"
+		go get -d "${podman_repo}" || true
+		pushd "${GOPATH}/src/${podman_repo}"
+		git checkout v"${version}"
+		make BUILDTAGS="selinux seccomp"
+		sudo -E PATH=$PATH make install PREFIX=/usr
```

Comment:
Note: Is the `PREFIX` required? Although we don't expect (or want) devs to run this script, if they did it might cause surprising results. Might be marginally safer to write to `/usr/local` or add a check on `KATA_DEV_MODE` (which we do use in various CI intsall scripts, but not the `setup_env_*` ones atm fwics).

Reply:
@jodh-intel I removed the PREFIX part